framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql

```diff
@@ -1,3 +1,5 @@
-alter session set `planner.enable_broadcast_join`=false;
+set `planner.enable_broadcast_join`=false;
+set `planner.slice_target`=1;
 select * from hbase.customer1 c, cp.`tpch/orders.parquet` o where cast(c.orders.id as bigint) = o.o_orderkey and c.orders.id <= '200';
-alter session set `planner.enable_broadcast_join`=true; 
+reset `planner.slice_target`;
+set `planner.enable_broadcast_join`=true;
```

Comment:
Can we confirm before run the planner.enable_broadcast_join was indeed set to true?

Reply:
Addressed comments. 
Using reset for planner.enable_broadcast_join

Reply:
Did you actually use reset?